src/main/scala/is/hail/stats/Distributions.scala

```diff
@@ -0,0 +1,66 @@
+package is.hail.stats
+
+import breeze.stats.distributions._
+import org.apache.commons.math3.random.JDKRandomGenerator
+import is.hail.utils._
+
+sealed trait Distribution {
+    def getBreezeDist(seed: Int): Rand[Double]
+}
+case class UniformDist(minVal: Double, maxVal: Double) extends Distribution{
+  if (minVal >= maxVal)
+    fatal("minVal must less than maxVal")
+
+
+
+  override def getBreezeDist(seed: Int): Rand[Double] = {
+    val generator = new JDKRandomGenerator
+    generator.setSeed(seed)
+    val basis = new RandBasis(generator)
+
+    new Uniform(minVal, maxVal)(basis)
+  }
+}
+
+case class BetaDist(a: Double, b: Double) extends Distribution {
+
+  override def getBreezeDist(seed: Int): Rand[Double] = {
+    val generator = new JDKRandomGenerator
+    generator.setSeed(seed)
+    val basis = new RandBasis(generator)
+
```

Comment:
Would it make more sense to have getBreezeDist take a RandBasis directly?  Then you can pass in the perVariantRandomBasis rather than construct another one, and remove the code duplicated above, right?